src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -63,7 +63,9 @@ def rotate_every_two(x):
 
 
 def apply_rotary_pos_emb(x, sincos, offset=0):
-    sin, cos = map(lambda t: t[None, offset : x.shape[1] + offset, None, :].repeat_interleave(2, 3), sincos)
+    sin, cos = map(
+        lambda t: t.view(-1, 1).repeat(1, 2).view(t.shape[0], -1)[None, offset : x.shape[1] + offset, None, :], sincos
+    )
```

Comment:
Nice!

(nit) this looks a bit too magical, maybe split this on multiple lines and add a comment explaining what it's doing.